src/stylecop.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
If you use a `.ruleset` file, I don't think you need this.

Reply:
See prior comments.